book.rb

```diff
@@ -0,0 +1,20 @@
+# Public: Book class used to create book object
+#
+# All methods are class methods and should be called on the Book class.
+class Book
+  attr_reader :rental
```

Comment:
-  [ ] Book class should have _title_ and _author_ instance set as _Getters_ and _Setters_:
```ruby
    attr_accessor :title, :author